CHANGELOG.md

```diff
@@ -20,6 +20,8 @@ A `secondaryLevel: true` option is available to add operations to the widget's c
 * Fixes all eslint warnings.
 * When pasting a widget from the clipboard, the correct widget type is always offered on the "Add Content" menu.
 * Widget live preview is now attempting to auto-position the Widget Editor modal only if no explicit widget configuration (`options.origin`) is provided.
+* `required` is now implemented on the server side as well for `relationship` fields. It behaves like `min: 1`. It was always implemented on the front end. However, note that a relationship can still become empty if the related document is archived or deleted.
```

Comment:
Without this change, widget live preview would show an empty widget if you x'd out the image in the widget editor, although you were not allowed to save that way. Now it shows the last valid state it got from the editor, as it does for all other invalid input.